### DISCUSSION TITLE
Welcome to Vue RFCs Discussions!

### DISCUSSION THREAD

**Discussion by yyx990803** [vuejs/rfcs#245] [General]
## 👋 Welcome!

  The Vue RFCs Discussion board's primary purpose is to provide a space for community members to provide feedback to submitted RFCs.

  Aside from formal RFCs, this place can also be used for:
  
  - Official announcements from the Vue.js team
  - Discussing rough ideas of new features before working on a formal RFC
  - Sharing thoughts on the future directions of the framework

  Note that this is **NOT** the place for:

  - Getting help on general usage questions
  - Discussing technical details of bugs
  - Non-official Vue projects

  In short, **the topics should be focused on what should be added to the framework**. Please be respectful in discussions, and try to focus on technical merits instead of personal preference.
